app/src/main/res/layout/schedule_day.xml

```diff
@@ -1,13 +0,0 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:orientation="horizontal">
-
-    <TextView
-        android:id="@+id/schedule_time"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center"
-        android:text="Hello World!"
-        android:textSize="18dp" />
-</LinearLayout>
```

Comment:
Why? I don't want to go deeper into code, so could you explain your changes briefly? Did you remove schedule_day?